lib/tpm2_hmac_auth.c

```diff
@@ -31,9 +31,78 @@
 
 #include <tss2/tss2_sys.h>
 
+#include <openssl/bio.h>
+#include <openssl/buffer.h>
+#include <openssl/evp.h>
+#include <openssl/sha.h>
```

Comment:
Can you please check that this builds with both OpenSSL 1.1.0 and OpenSSL < 1.1.0 ? It's annoying but there were a lot of API changes so we needed #ifdefery in other part of the tools.

Reply:
@idesai you forgot to answer this question.

Reply:
@martinezjavier. From looking at the change notes, The specific openssl api calls made here do not need the #ifdefery like other tools do.